packages/analytics-js-integrations/src/integrations/FacebookPixel/browser.js

```diff
@@ -294,7 +290,6 @@ class FacebookPixel {
           value: revValue,
         });
       }
```

Comment:
can we add info log stating `Not standard event & no custom mapping available` ?

Reply:
if none of the conditions pass through